src/node/interfaces.cpp

```diff
@@ -168,29 +168,25 @@ class NodeImpl : public Node
     bool getNodesStats(NodesStats& stats) override
```

Comment:
Not specifically related to this PR (I guess): Wouldn't be better if we return `NodesStats` here rather than a bool?